TileFormats/Instanced3DModel/README.md

```diff
@@ -22,65 +22,180 @@ A tile is composed of a header section immediately followed by a body section.
+### Instance Scaling
 
-The batch table is a `UTF-8` string containing JSON.  It immediately follows the header.  It can be extracted from the arraybuffer using the `TextDecoder` JavaScript API and transformed to a JavaScript object with `JSON.parse`.
+Scaling can be applied to instances using the `SCALE` and `SCALE_NON_UNIFORM` semantics.
+`SCALE` applies a uniform scale along all axes, and `SCALE_NON_UNIFORM` applies scaling to the `x`, `y`, and `z` axes independently.
 
-Each property in the object is an array with its length equal to `header.batchLength`.  Array elements can be any valid JSON data type, including objects and arrays.  Elements may be `null`.
+If `SCALE` and `SCALE_NON_UNIFORM` are defined for an instance, both scaling operations will be applied.
+
+### Examples
+
+#### Positions Only
+
+In this minimal example, we place 4 instances on the corners of a unit length square with the default orientation.
 
-An instance's `batchId` is used to access elements in each array and extract the corresponding properties.  For example, the following batch table has properties for two instances:
 ```json
 {
-    "id" : ["unique id", "another unique id"],
-    "displayName" : ["Tree species", "Another tree species"],
-    "yearPlanted" : [1999, 2003],
-    "location" : [{"x" : 1, "y" : 2}, {"x" : 3, "y" : 4}]
+    "INSTANCES_LENGTH" : 4,
+    "POSITION" : [
+        0.0, 0.0, 0.0, 
+        1.0, 0.0, 0.0, 
+        0.0, 0.0, 1.0, 
+        1.0, 0.0, 1.0
+    ]
 }
 ```
 
-The properties for the instance with `batchId = 0` are
-```javascript
-id[0] = 'unique id';
-displayName[0] = 'Tree species';
-yearBuilt[0] = 1999;
-location[0] = {x : 1, y : 2};
-```
+#### Quantized Positions and Oct-Encoded Normals
 
-The properties for `batchId = 1` are
-```javascript
-id[1] = 'another unique id';
-displayName[1] = 'Another tree species';
-yearBuilt[1] = 2003;
-location[1] = {x : 3, y : 4};
+In this example, the 4 instances will be placed with an orientation `up` of `[0.0, 1.0, 0.0]` and `right` of `[1.0, 0.0, 0.0]` in oct-encoded format 
+and they will be placed on the corners of a quantized volume that spans from `-250.0` to `250.0` units in the `x` and `z` directions.
+
+```json
+{
+    "INSTANCES_LENGTH" : 4,
+    "QUANTIZED_VOLUME_OFFSET" : [-250.0, 0, -250],
+    "QUANTIZED_VOLUME_SCALE" : [500.0, 0, 500.0],
+    "POSITION_QUANTIZED" : [
+        0, 0, 0,
+        65535, 0, 0,
+        0, 0, 65535,
+        65535, 0, 65535
+    ],
+    "NORMAL_UP_OCT32P" : [
+        32768, 65535,
+        32768, 65535,
+        32768, 65535,
+        32768, 65535
+    ],
+    "NORMAL_RIGHT_OCT32P" : [
+        65535, 32768,
+        65535, 32768,
+        65535, 32768,
+        65535, 32768
+    ]
+}
 ```
 
+## Batch Table
+
+Contains metadata organized by `batchId` that can be used for declarative styling.
+
+See the [Batch Table](https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/TileFormats/Batched3DModel#batch-table) reference for more information.
```

Comment:
Use relative links for links inside the repo.

Also add a `TODO` to change this link to the generic batch table.